packages/server/src/provisioning/stripe.ts

```diff
@@ -493,7 +493,7 @@ export class StripeProvisioner extends BasicProvisioner {
                 undefined,
                 undefined,
                 true,
-                "history"
+                "History"
```

Comment:
This just seemed like a typo.

Reply:
That argument is case-insensitive. So either way is fine :)